lib/elastic_record/callbacks.rb

```diff
@@ -60,7 +60,11 @@ def elastic_search_value(field, mapping)
         when :nested
           value.map(&:as_search)
         else
```

Comment:
So, your attribute definition looks like:

```
  "manual_geocoordinate": {
    "type": "geo_point",
    "class_name": "GeoPoint",
    "attributes": {
      "lat": {
        "type": "float"
      },
      "lon": {
        "type": "float"
      }
    }
  },
```

And then [axle_attributes](https://github.com/data-axle/axle_attributes/blob/4c130518b993d40c6de7ea79dfa5b988e7ba80d6/lib/axle_attributes/definition.rb) would translate that into elastic_record/superstore-speak?